pages.zh/osx/netstat.md

```diff
@@ -0,0 +1,27 @@
+# netstat
+
+> 显示与网络相关的信息，如打开的连接、打开的套接字端口等.
+
+- 列出所有端口:
+
+`netstat -a`
+
+- 列出所有被侦听端口:
+
+`netstat -l`
+
+- 列出侦听的TCP端口:
+
+`netstat -t`
+
+- 显示监听给定协议监听的PID和程序名:
+
+`netstat -p {协议}`
```

Comment:
```suggestion
`netstat -p {{协议}}`
```

Reply:
Eeee.. What should I do next?

Reply:
@wizarot Just click "commit suggestion", and change the message to `netstat: add missing curly braces`. It's perfectly fine in this instance to have an extra commit for it, because it was actually a separate commit on the English page too IIRC :-)

Reply:
I click "commit suggestion" and what should I do next?
Sorry, I have no experience in this field.